### PR TITLE
PYTHON-266: Add DISTINCT support

### PR DESCRIPTION
**Pull request by aboudreault**

### CODE REVIEW

File: cassandra/cqlengine/statements.py [hunk 1]
```diff
@@ -550,6 +550,7 @@ class SelectStatement(BaseCQLStatement):
     def __init__(self,
                  table,
                  fields=None,
+                 distinct_fields=[],
```
Comment (aholmberg) [inline cassandra/cqlengine/statements.py]: Curious about why this was added in the middle of the args. I think this is probably okay as an internal API, but I usually only append keyword arguments where possible.

Comment (aholmberg) [inline cassandra/cqlengine/statements.py]: Would it be okay to default to None? Looks safe here, but mutable defaults are usually discouraged.

Comment (aboudreault) [inline cassandra/cqlengine/statements.py]: no objection to move. There is not reason really.

Comment (aboudreault) [inline cassandra/cqlengine/statements.py]: +1 for the None.

**Comment by aholmberg** [comment]
+1

**Comment by aboudreault** [comment]
Comments have been addressed.

**Pull request merged**